cosinnus/models/conference.py

```diff
@@ -665,11 +665,15 @@ def sync_rocketchat_room(self, force=False):
         if settings.COSINNUS_ROCKET_ENABLED and self.type in self.ROCKETCHAT_ROOM_TYPES \
                 and not settings.COSINNUS_CONFERENCES_USE_COMPACT_MODE:
             if not self.rocket_chat_room_id or force:
-                from cosinnus_message.rocket_chat import RocketChatConnection # noqa
-                rocket = RocketChatConnection()
-                room_name = f'{self.slug}-{self.group.slug}-{get_random_string(7)}'
-                internal_room_id = rocket.create_private_room(room_name, self.creator, 
-                      member_users=self.group.actual_members, additional_admin_users=self.group.actual_admins)
+                from cosinnus_message.rocket_chat import RocketChatConnection, RocketChatDownException # noqa
+                try:
+                    rocket = RocketChatConnection()
+                    room_name = f'{self.slug}-{self.group.slug}-{get_random_string(7)}'
+                    internal_room_id = rocket.create_private_room(room_name, self.creator,
+                          member_users=self.group.actual_members, additional_admin_users=self.group.actual_admins)
+                except RocketChatDownException:
```

Comment:
(this counts for all placed where we catch `RocketChatDownException`):
Should we maybe also add an additional blanket catch on `Exception` to make all RC calls fully safe from bubbling up to the users on an exception? Or are they handled in the deper code always, so that `RocketChatDownException` is the only exception that could possibly bubble?

Reply:
Sounds good. I pushed an commit handling the basic `Exception.` everywhere. Also added a new user message to be displayed in that case, as we discussed. Feel free to change the wording.